brave/brave-core
# feat(wallet): Switch over to Updated Nav

## Description 
Switched over to the new `Buy/Send/Swap/Deposit` nav.


Resolves <https://github.com/brave/brave-browser/issues/27008>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

### Testing Desktop
1. Go to the `Portfolio` page,
2. Click on `Send`, it should nav to brave://wallet/send
3. On the left hand side of the `Send` tab, click on `Swap`, it should nav to brave://wallet/swap
4. Test all nav buttons in the the nav, they should nav to the correct screens.
5. note(Deposit Funds and Fund Wallet screen will not have this nav to start)

https://user-images.githubusercontent.com/40611140/204326872-d93000b0-3696-4c49-b966-6852e4ec9374.mov

### Testing Panel
1. Open the `Panel`
2. Click on `Send`, it should nav to brave://wallet/send
3. Click on `Swap`, it should nav to brave://wallet/swap
4. Click on `Buy`, it should nav to brave://wallet/crypto/fund-wallet
5 Click on `Deposit`, it should nav to brave://wallet/crypto/deposit-funds

https://user-images.githubusercontent.com/40611140/204326922-6e8a351f-dc4f-44e1-9a89-838257bcca18.mov
